coffit/__init__.py

```diff
@@ -1,8 +1,7 @@
+"""Init."""
+
 import socketio
 
 sio = socketio.Server()
 
-from . import models  # noqa
-from . import api, auth  # noqa
```

Comment:
I removed because there is some circular importing: https://github.com/slint/coffit/blob/base_app/coffit/api.py#L1